lib/will_paginate/view_helpers/action_view.rb

```diff
@@ -115,7 +115,8 @@ def url(page)
         url_params = @base_url_params.dup
         add_current_page_param(url_params, page)
 
-        @template.url_for(url_params)
+				h = Rails.application.routes.url_helpers
```

Comment:
Right now this breaks tests because we don't run a whole Rails app in the test suite. In fact, will_paginate supports Action View being used on its own, therefore it should  always work without a `Rails.application` being defined. I'm not convinced that this change is a good idea.

Reply:
Apologies, I opened this PR by mistake.